crates/runtime/src/vmcontext.rs

```diff
@@ -778,16 +778,81 @@ impl VMContext {
 /// This is provided for use with the `Func::new_unchecked` and
 /// `Func::call_unchecked` APIs. In general it's unlikely you should be using
 /// this from Rust, rather using APIs like `Func::wrap` and `TypedFunc::call`.
+///
+/// This is notably an "unsafe" way to work with `Val` and it's recommended to
```

Comment:
*ValRaw

Reply:
The comment here is actually intended to point to `Val` in that `ValRaw` is the unsafe way of working with what is otherwise a wasm `Val` value.